IBX-5100: Reorganize Admin Panel/Content model doc (#1920)
ezsystems/developer-documentation: docs/administration/admin_panel/corporate_admin_panel.md

---

> description: You can manage Corporate Accounts in the Admin Panel.

Reviewer: Is this the name we ultimately want as feature name? @juskora @MagdalenaZuba

Reviewer 2: We were urged not to use Corporate Accounts as a name of the feature, instead we can use business accounts/companies profiles/users with customer type. Unfortunately, the UI clearly states Corporate Accounts, so I will rely on @juskora expertise is that left on purpose or by accident.

Reviewer 3: Actually I have no knowledge if it's a leftover or not, so I thin it is safe to use wording described by Magda in a comment earlier.

Reviewer 2: You can also consult Bertrand as he is PM responsible for Customer Portal

---

# Corporate

You can manage Corporate Accounts in the Admin Panel.

> There, in the **Corporate** section, you can find find basic information about exisiting **Companies**, for example, Details, Versions, Locations, Translations, a list of members, billing addresses and technical details regarding the organization such as visibility, IDs, relations and so on.

Reviewer: Here and later, please split paragraphs into shorter lines, ideally as sentence or clause end.

> There, in the **Corporate** section, you can find find basic information about exisiting **Companies**, for example, Details, Versions, Locations, Translations, a list of members, billing addresses and technical details regarding the organization such as visibility, IDs, relations and so on.

Reviewer: ```suggestion
There, in the **Corporate** section, you can find basic information about existing companies, for example, details, versions, Locations, translations, a list of members, billing addresses and technical details regarding the organization, such as visibility, IDs, or relations.
```
Companies with regular text, because we bold UI labels.

![Corporate Account](admin_panel_corporate.png "Corporate Account")

> See [Customer Management](https://doc.ibexa.co/projects/userguide/en/latest/customer_management/manage_customers/) for further information. 

Reviewer: ```suggestion
See [Customer management](https://doc.ibexa.co/projects/userguide/en/latest/customer_management/manage_customers/) for further information. 
```
